### DISCUSSION TITLE
Ignore not working with `check`

### DISCUSSION THREAD

**Discussion by viveleroi** [biomejs/biome#1617] [Q&A]
My .gitignore has an entry for `/.next/`, and I've read that Biome is supposed to read the gitignore file. Yet the files inside .next are still checked.

I added an entry to the config, and yet the .next directory is still checked.

```
{
  "$schema": "https://biomejs.dev/schemas/1.5.2/schema.json",
  "organizeImports": {
    "enabled": true
  },
  "linter": {
    "ignore": [".next"],
    "rules": {
      "recommended": true,
      "nursery": {
        "noUnusedImports": "error"
      }
    }
  },
  "formatter": {
    "ignore": [".next"],
    "indentStyle": "space",
    "lineWidth": 120
  }
}
```

**Comment by ematipico** [comment]
> I've read that Biome is supposed to read the gitignore file

You have to enable the feature: https://biomejs.dev/guides/integrate-in-vcs/

> I added an entry to the config, and yet the .next directory is still checked.

Can you share how you run Biome from the CLI?

**Reply by viveleroi** [reply]
Using the vcs integration seems to work as expected. Thanks

As for the cli, I tried both `pnpm dlx @biomejs/biome check .` and a package.json script `"check": "biome check ."` and then using `pnpm run check`

**Comment by Conaclos** [comment]
I tested your configuration using pnpm. Biome correctly ignores `.next`.
I executed the following commands:

```shell
❯ vim biome.json # add biome.json

❯ cat biome.json
{
  "$schema": "https://biomejs.dev/schemas/1.5.2/schema.json",
  "organizeImports": {
    "enabled": true
  },
  "linter": {
    "ignore": [".next"],
    "rules": {
      "recommended": true,
      "nursery": {
        "noUnusedImports": "error"
      }
    }
  },
  "formatter": {
    "ignore": [".next"],
    "indentStyle": "space",
    "lineWidth": 120
  }
}

❯ pnpm init

❯ pnpm add @biomejs/biome

❯ vim package.json # edit package.json

❯ cat package.json
{
  "name": "next",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "scripts": {
    "check": "biome check ."
  },
  "keywords": [],
  "author": "",
  "license": "ISC",
  "devDependencies": {
    "@biomejs/biome": "^1.5.2"
  }
}

❯ tree -a
.
├── .next
│   └── index.js
├── biome.json
├── node_modules
│   └── ...
├── package.json
└── pnpm-lock.yaml

❯ pnpm check
Checked 2 file(s) in 596µs

```

**Comment by Conaclos** [comment]
Closing because the issue should be solved.

**Comment by arshad-yaseen** [comment]
Any updates?

**Reply by ematipico** [reply]
It was a bug and it should be fixed.
